gen/DocumentFormat.OpenXml.Generator.Linq/DocumentFormat.OpenXml.Generator.Linq.csproj

```diff
@@ -30,4 +30,12 @@
     </Otherwise>
   </Choose>
 
+  <ItemGroup>
+    <PackageReference Include="CommandLineParser" Version="2.8.0" />
```

Comment:
Can you use [System.CommandLine](https://www.nuget.org/packages/System.CommandLine/2.0.0-beta4.22272.1)?  It's almost at v1 and shouldn't have too many changes (I'll react to them if needed)